templates/appliance/hardware.html

```diff
@@ -0,0 +1,693 @@
+            <div class="u-align--center">
+              {{
+                image(
+                  url="https://assets.ubuntu.com/v1/270058f5-Intel-NUC7i3DNHE-Box3.jpg?w=225",
+                  alt="",
+                  height="106",
+                  width="225",
+                  hi_def=True,
+                  loading="auto",
+                  attrs={"style": "margin: 1rem 0;"},
+                ) | safe
+              }}
+            </div>
+          </div>
+          <div class="p-card__footer">
+            <a href="https://www.intel.co.uk/content/www/uk/en/products/boards-kits/nuc/mini-pcs.html" class="p-button--positive u-no-margin--bottom"><span class="p-link--external">Buy it</span></a>
+          </div>
+        </div>
+      </div>
+      <div class="row u-equal-height">
+        <div class="col-4 p-card--layout">
+          <div class="p-card__header">
+            <h4>Intel NUC7i5DNHE</h4>
+            <div class="p-label--new">CERTIFIED</div>
+          </div>
+          <div class="p-card__main">
+            <div class="u-align--center">
+              {{
+                image(
+                  url="https://assets.ubuntu.com/v1/d45cf681-Intel-NUC7i5DNHE-Box4.jpg?w=225",
+                  alt="",
+                  height="106",
+                  width="225",
+                  hi_def=True,
+                  loading="auto",
+                  attrs={"style": "margin: 1rem 0;"},
+                ) | safe
+              }}
+            </div>
+          </div>
+          <div class="p-card__footer">
+            <a href="https://www.intel.co.uk/content/www/uk/en/products/boards-kits/nuc/mini-pcs.html" class="p-button--positive u-no-margin--bottom"><span class="p-link--external">Buy it</span></a>
+          </div>
+        </div>
+        <div class="col-4 p-card--layout">
+          <div class="p-card__header">
+            <h4>Intel TANK-870-Q170</h4>
+            <div class="p-label--new">CERTIFIED</div>
+          </div>
+          <div class="p-card__main">
+            <div class="u-align--center">
+              {{
+                image(
+                  url="https://assets.ubuntu.com/v1/139526f0-Intel-TANK-870-Q170-Box5.jpg?h=150",
+                  alt="",
+                  height="150",
+                  width="143",
+                  hi_def=True,
+                  loading="auto"
+                ) | safe
+              }}
+            </div>
+          </div>
+          <div class="p-card__footer">
+            <a href="https://www.intel.co.uk/content/www/uk/en/products/boards-kits/nuc/mini-pcs.html" class="p-button--positive u-no-margin--bottom"><span class="p-link--external">Buy it</span></a>
+          </div>
+        </div>
+      </div>
+
+      <div class="u-fixed-width">
+        <hr class="p-separator" style="margin: 2rem 0;" />
+      </div>
+
+      <div class="row">
+        <h3 class="u-sv3">Certified appliances for Intel NUC</h3>
+      </div>
+
+      <div class="row">
```

Comment:
this could be an include for _featured_appliances

Reply:
Done